registries/adapters/openshift_adapter.go

```diff
@@ -42,7 +42,11 @@ type OpenShiftImage struct {
 
 // RegistryName - Retrieve the registry name
 func (r OpenShiftAdapter) RegistryName() string {
-	return strings.TrimPrefix(r.Config.URL.String(), "https://")
+	var name string
+	name = strings.TrimPrefix(r.Config.URL.String(), "https://")
+	name = strings.TrimPrefix(name, "http://")
+
+	return name
```

Comment:
I'm going to assume this url can be ANY valid url. If that's the case I'd rather strip off any leading `://` that comes before it. So I'd rather do something like this:

```golang
func (r OpenShiftAdapter) RegistryName() string {
	name := strings.Split(r.Config.URL.String(), "://")

	if len(name) > 1 {
		return name[1]
	}
	return name[0]
```

This will cover all URLs `file://`, `http://`, `ftp://`, `https://`.